content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx

```diff
@@ -1,17 +1,39 @@
 ---
 title: 'Examples'
-metaTitle: 'Prisma Client Extensions | Examples'
-metaDescription: 'Extend the functionality of Prisma Client: Examples'
+metaTitle: 'Prisma Client extensions | Prisma & Community-built extensions and examples'
+metaDescription: 'Prisma Client extensions | Prisma & Community-built extensions and examples'
 ---
 
-<TopBlock>
+## Prisma-built extensions
 
-The extensions are provided as examples only, and without warranty. We recommend using the examples as a source of inspiration for building your own extensions.
+The following is a list of extensions we've built at Prisma:
+
+| Extension                                                                                             | Description                                                                                                               |
+| :---------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [`@prisma/extension-accelerate`](https://www.prisma.io/docs/data-platform/accelerate/getting-started) | Enables Accelerate, a global database cache                                                                               |
```

Comment:
I think we have a better description for accelerate?

Reply:
@Jolg42 I've added a suggestion below that should suffice for a simple description.